### PR TITLE
[RHPAM-3056] Business Central remains "Container Creating" if missing known_hosts file in the githooks configuration

### PR DESCRIPTION
**Pull request by desmax74**
https://issues.redhat.com/browse/RHPAM-3056
Improved description

@spolti @jakubschwan @mramendi

### CODE REVIEW

**Review by jakubschwan — COMMENTED** [review]
Can you please also update description for KieApp as well? We should keep it in sync. Updated description in KieApp should be also visible in OCP yaml editor.

**Comment by desmax74** [comment]
@jakubschwan kieapp crd updated

File: deploy/crds/kieapp.crd.yaml [hunk 1]
```diff
@@ -497,8 +497,8 @@ spec:
                               be mounted.
                             type: string
                           sshSecret:
-                            description: Secret to use for ssh key and known hosts
-                              file.
+                            description: 'Secret to use for ssh key and known hosts
+                              file. The secret must contain two files: id_rsa and known_hosts'
```
Comment (tchughesiv) [inline deploy/crds/kieapp.crd.yaml:501]: make that change here instead and then run `make test` to generate the crd.
https://github.com/kiegroup/kie-cloud-operator/blob/8a00102178cf17c2f23266e994638eab485947d8/pkg/apis/app/v2/kieapp_types.go#L303

**Review by tchughesiv — CHANGES_REQUESTED** [review]

**Review by tchughesiv — CHANGES_REQUESTED** [review]
lgtm ... squash your commits and we can merge.

**Comment by desmax74** [comment]
@tchughesiv squash done

**Review by tchughesiv — APPROVED** [review]

**Review by jakubschwan — APPROVED** [review]

**Pull request merged**